src/ck-core/charm++.h

```diff
@@ -234,10 +234,11 @@ class Chare {
 #ifndef CMK_CHARE_USE_PTR
     int chareIdx;                  // index in the chare obj table (chare_objs)
 #endif
+    int r_count;
```

Comment:
rename this more descriptively or add a comment describing its use

Reply:
Has been removed.